uwblueprint/sistering
# Add Edit Volunteer Row

## Ticket link
Closes #175



## Implementation description
* Added the avatar icon as svg
* Created AdminScheduleVolunteerRow component


## Steps to test
1. Go to `frontend/src/components/pages/Default.tsx` and add the following lines of code:
```js
// directly before line 13
import AdminScheduleVolunteerRow from "../admin/schedule/AdminScheduleVolunteerRow";

// directly before line 170
<AdminScheduleVolunteerRow
  volunteerID="1"
  volunteerName="Amanda Du"
  note=""
  isConfirmed={false}
/>
<AdminScheduleVolunteerRow
  volunteerID="2"
  volunteerName="Soha Khan"
  note="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam tincidunt ex sed orci malesuada, vel dignissim erat malesuada. Ut nec urna eget nisl tincidunt vestibulum. Nam in feugiat ligula. Donec nec egestas arcu, at dapibus magna. Phasellus non ligula auctor, tincidunt mi vel, tincidunt ligula. Mauris rhoncus id mi vitae tempus.."
  isConfirmed
/>
```
2. `docker-compose up --build`
3. Navigate to http://localhost:3000/
4. Verify the AdminScheduleVolunteerRow looks and works as expected



## What should reviewers focus on?
* Verify the AdminScheduleVolunteerRow looks and works as expected


## Checklist
